### ISSUE TITLE
[Tooltip] Content is duplicated in the DOM

### ISSUE THREAD

**Issue by joseerobless** [radix-ui/primitives#3689]
### Current Behavior

The Tooltip component is duplicating content in the DOM, creating a usability issue since users, using screen readers, will hear it twice when navigating with arrow keys.

### Expected behavior

Tooltip content should only be rendered once

### Reproducible example

[CodeSandbox](https://www.radix-ui.com/primitives/docs/components/tooltip)

### Suggested solution

Make it so the content is only in the DOM once or if absolutely necessary make the duplicated content not accessible by screen readers.

### Additional context

<img width="1202" height="735" alt="Image" src="https://github.com/user-attachments/assets/20105410-3ce0-4c2d-a1d2-e644353532c0" />

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Radix Package(s) |         |         |
| React            | n/a     |         |
| Browser          |         |         |
| Assistive tech   |         |         |
| Node             | n/a     |         |
| npm/yarn/pnpm    |         |         |
| Operating System |         |         |

**Comment by abhidatta0** [comment]
Fixed on PR #3767
